### PR TITLE
use XLStylizedEmpty in pivot table

### PR DESCRIPTION
**Pull request by cwuethrich**
At least don't throw an exception when loading a pivot table with borders. #1511

### CODE REVIEW

**Comment by cwuethrich** [comment]
Is it possible to include this change? in our case, we don't use the styling of the table. But if there is a style, we can't load the whole file.

**Comment by binn** [comment]
Any update on this merge?

**Review by jahav — COMMENTED** [review]
The core bug is that fluent API for border always expects container. 

Not all types that have a style use IStylized semantic (that is the type of the `_container`). IStylized expects a hierarchical structure with ranges, but pivot table styles are more like a set of conditions that apply differential styles to all cells in the pivot table. When user changes pivot table report, conditions are reevaluated and style the user sees changes.

So basically replace  `return _container.Style;` with `return _style;` in the XLBorder.

**Comment by jahav** [comment]
I can't add commits to a PR from a fork, I added them in #1928 with commit from this PR. File doesn't crash on load and I get some border properties.
